Podfile.lock

```diff
@@ -130,4 +130,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 02377777b693c2e84274e2a24ccfe6c9d5064b3c
 
-COCOAPODS: 1.10.0
+COCOAPODS: 1.10.1
```

Comment:
We should revert this change. :)

You may want to run `pod install` using `bundle exec pod install` to avoid your global `pod` being used.